src/TypeMatcher/Asset/Attribute/MediaLinkAssetAttributeTypeMatcher.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Synolia\SyliusAkeneoPlugin\TypeMatcher\Asset\Attribute;
+
+use Sylius\Component\Attribute\AttributeType\TextAttributeType;
+use Synolia\SyliusAkeneoPlugin\Builder\Asset\Attribute\MediaLinkAssetAttributeValueBuilder;
+
+final class MediaLinkAssetAttributeTypeMatcher implements AssetAttributeTypeMatcherInterface
+{
+    private const SUPPORTED_TYPE = 'media_link';
```

Comment:
IMHO, we should write that in the documentation, we only support `media_link` type .